### PR TITLE
Feature/display details pok

### PR DESCRIPTION
**Pull request by teddymeksavanh**

### CODE REVIEW

File: public/js/lib/pokedex/container/Carousel.js [hunk 1]
```diff
@@ -2,14 +2,20 @@ import {connect} from "react-redux";
...
+    setSelectedPokemonForDetails,
+    openDetails
 } from "../actions/carouselActions";
 
 const mapStateToProps = state => state;
 
 const mapDispatchToProps = dispatch => {
     return {
-
+      setSelectedPokemonForDetails(pokemon) {
```
Comment (dydyF05) [inline public/js/lib/pokedex/container/Carousel.js]: double indent mofo

File: public/js/lib/pokedex/container/Carousel.js [hunk 2]
```diff
@@ -2,14 +2,20 @@ import {connect} from "react-redux";
...
+        dispatch(setSelectedPokemonForDetails(pokemon));
+      },
+      openDetails() {
+        dispatch(openDetails());
```
Comment (dydyF05) [inline public/js/lib/pokedex/container/Carousel.js]: oh yeah

Comment (dydyF05) [inline public/js/lib/pokedex/container/Carousel.js]: ohhh yeah harder

File: public/js/lib/pokedex/ui/Carousel.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import Slider from 'react-slick';
 import Card from '../container/Card';
 import IconButton from 'material-ui/IconButton';
 import LocationSVG from 'material-ui/svg-icons/action/room';
+import DescriptionSVG from 'material-ui/svg-icons/action/description';
 
 const styles = {
   cardWrapper : {
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Carousel.js]: work it

File: public/js/lib/pokedex/ui/Carousel.js [hunk 4]
```diff
@@ -62,7 +63,19 @@ export default class Carousel extends React.PureComponent {
             <div className="card">
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Carousel.js:63]: make it

File: public/js/lib/pokedex/ui/Carousel.js [hunk 5]
```diff
@@ -62,7 +63,19 @@ export default class Carousel extends React.PureComponent {
...
               <span className="card-number">{this.props.pokemons.all[key].id_national}</span>
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Carousel.js]: do it

File: public/js/lib/pokedex/ui/Carousel.js [hunk 6]
```diff
@@ -62,7 +63,19 @@ export default class Carousel extends React.PureComponent {
...
               <img src={this.props.pokemons.all[key].icon} className="card-pokemon"/>
-              <IconButton style={styles.cardIconLocationWrapper} iconStyle={styles.cardIconLocation} tooltipPosition="top-center" tooltip="Position" children={<LocationSVG/>}/>
+              <IconButton
+                style={styles.cardIconLocationWrapper}
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Carousel.js]: make us

File: public/js/lib/pokedex/ui/Home.js [hunk 7]
```diff
@@ -15,11 +15,32 @@ import AddCircleOutlineSVG from 'material-ui/svg-icons/content/add-circle-outlin
 import PokeSearch from '../container/PokeSearch';
 import jQuery from 'jquery';
 import Screenfull from 'screenfull';
+import Close from 'material-ui/svg-icons/action/highlight-off';
 
-const styles = {
+const styles = {
+  buttonClose : {
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js]: harder

File: public/js/lib/pokedex/ui/Home.js [hunk 8]
```diff
@@ -15,11 +15,32 @@ import AddCircleOutlineSVG from 'material-ui/svg-icons/content/add-circle-outlin
...
+    position: 'absolute',
+    top: '0',
+    right: '0',
+    margin: '15px',
+    height: '100px',
+    width: '100px'
+  },
+  iconClose : {
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js]: better

File: public/js/lib/pokedex/ui/Home.js [hunk 9]
```diff
@@ -15,11 +15,32 @@ import AddCircleOutlineSVG from 'material-ui/svg-icons/content/add-circle-outlin
...
+    color: 'white',
+    height: '80px',
+    width: '80px'
+  },
   icon: {
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js]: faster

File: public/js/lib/pokedex/ui/Home.js [hunk 10]
```diff
@@ -15,11 +15,32 @@ import AddCircleOutlineSVG from 'material-ui/svg-icons/content/add-circle-outlin
...
     height: '30px',
     color: 'white'
+  },
+  pokemonName: {
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js]: stronger

File: public/js/lib/pokedex/ui/Home.js [hunk 11]
```diff
@@ -73,11 +94,44 @@ export default class Home extends React.PureComponent {
       }
     }
 
+    renderPokemonDetails() {
+      if(this.props.carousel.showDetails) {
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js:98]: hey hey

File: public/js/lib/pokedex/ui/Home.js [hunk 12]
```diff
@@ -73,11 +94,44 @@ export default class Home extends React.PureComponent {
...
+        return (
+          <div className="card-details align full-height full-width">
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js]: my myyyyy

File: public/js/lib/pokedex/ui/Home.js [hunk 13]
```diff
@@ -73,11 +94,44 @@ export default class Home extends React.PureComponent {
...
+                <ul>
+                  <li className="text-center">
+                    <span style={styles.pokemonId}>No. {this.props.carousel.selectedPokemonForDetails.id_national}</span>
+                  </li>
+                  <li className="text-center">
+                    <span style={styles.pokemonName}>{this.props.carousel.selectedPokemonForDetails.name}</span>
+                  </li>
+                  <li>
+                  </li>
+                  <li>
```
Comment (dydyF05) [inline public/js/lib/pokedex/ui/Home.js]: rock n roll can never diiiie

**Review by dydyF05 — CHANGES_REQUESTED** [review]

File: public/js/lib/pokedex/ui/Carousel.js (hunk 3 above)
Comment (teddymeksavanh) [reply public/js/lib/pokedex/ui/Carousel.js]: work it?

**Pull request merged**